xrmocap/core/evaluation/metrics/mpjpe_metric.py

```diff
@@ -11,27 +11,112 @@
 
 
 class MPJPEMetric(BaseMetric):
-    """Mean per-joint position error(MPJPE)."""
-    RANK = 0
+    """Mean per-joint position error(MPJPE).
+
+    This is a rank-1 matric, depends on rank-0 matric PredictionMatcher. If the
```

Comment:
```suggestion
    This is a rank-1 metric, depends on rank-0 metric PredictionMatcher. If the
```
7 `matric` found in this webpage.

Reply:
Fixed in [3bb41b5](https://github.com/openxrlab/xrmocap/pull/79/commits/3bb41b545e5ec89d8455a37f9f15d990158608c6).